python/lsst/ts/envsensors/device/vcp_ftdi.py

```diff
@@ -68,39 +70,43 @@ def __init__(
             auto_detach=False,
         )
 
-    def init(self) -> None:
-        """Initialize the Sensor Device."""
-        pass
-
     async def basic_open(self) -> None:
         """Open the Sensor Device.
 
+        Opens the virtual communications port and flushes the device input and
+        output buffers.
+
         Raises
         ------
         IOError if virtual communications port fails to open.
         """
         self._vcp.open()
+        self._vcp.baudrate = 19600
         if not self._vcp.closed:
             self._log.debug("VCP open.")
             self._vcp.flush()
         else:
-            self._log.debug("Failed to open VCP.")
+            self._log.error("Failed to open VCP.")
```

Comment:
Please consider spelling out "VCP".

Also...is it important to make the log a private attribute, instead of the usual `self.log`?

Reply:
Done